### PR TITLE
Converted first half of MPI_C* (from MPI_Cancel to MPI_Close_port)

### PR DESCRIPTION
**Pull request by Colton-K**
Signed-off-by: Colton Kammes <ckammes@nd.edu>

### CODE REVIEW

**Comment by jjhursey** [comment]
ok to test

File: ompi/mpi/man/man3/MPI_Cart_map.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+Before the error value is returned, the current MPI error handler is
+called. By default, this error handler aborts the MPI job, except for
+I/O function errors. The error handler may be changed with
+`MPI_Comm_set_errhandler`; the predefined error handler `MPI_ERRORS_RETURN`
+may be used to cause error values to be returned. Note that MPI does not
+guarantee that an MPI program can continue past an error.
+
+# See Also
+
+[`MPI_Graph_map``(3)](MPI_Graph_map`.html)
```
Comment (jsquyres) [inline ompi/mpi/man/man3/MPI_Cart_map.md:79]: Watch the double qouting here.

File: ompi/mpi/man/man3/MPI_Cart_rank.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+Before the error value is returned, the current MPI error handler is
+called. By default, this error handler aborts the MPI job, except for
+I/O function errors. The error handler may be changed with
+`MPI_Comm_set_errhandler`; the predefined error handler `MPI_ERRORS_RETURN`
+may be used to cause error values to be returned. Note that MPI does not
+guarantee that an MPI program can continue past an error.
+
+# See Also
+
+[`MPI_Cart_create``(3)](MPI_Cart_create`.html)
```
Comment (jsquyres) [inline ompi/mpi/man/man3/MPI_Cart_rank.md:72]: Double quoting

File: ompi/mpi/man/man3/MPI_Cart_rank.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+* `IERROR` : Fortran only: Error status (integer).
+
+# Description
+
+For a process group with Cartesian structure, the function `MPI_Cart_rank`
+translates the logical process coordinates to process `rank`s as they are
+used by the point-to-point routines. For dimension i with periods(i) =
+true, if the coordinate, `coords`(i), is out of range, that is, `coords`(i)
+< 0 or `coords`(i) >= dims(i), it is shifted back to the interval 0 =<
+`coords`(i) < dims(i) automatically. Out-of-range coordinates are
```
Comment (jsquyres) [inline ompi/mpi/man/man3/MPI_Cart_rank.md:56]: Note that this paragraph actually chose to express itself in Fortran notation -- `periods(i)` and `coords(i)` and `dims(i)` (vs. `periods[i]` and `coords[i]` and `dims[i]`).

Can you either include the `(i)` in the quoting, or convert it to C-style and include the `[i]` in the quoting?

File: ompi/mpi/man/man3/MPI_Cart_shift.md [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+`MPI_PROC_NULL` may be returned in ``rank_source`` or ``rank_dest``, indicating
+that the source or the destination for the shift is out of range.
+
+Example: The `comm`unicator, `comm`, has a two-dimensional, periodic,
+Cartesian topology associated with it. A two-dimensional array of REALs
+is stored one element per process, in variable A. One wishes to skew
+this array, by shifting column i (vertically, i.e., along the column) by
+i steps.
+  
+  ...
```
Comment (jsquyres) [inline ompi/mpi/man/man3/MPI_Cart_shift.md:83]: This is actually Fortran code.  Can you triple-single-quote it?

Change the "C" to "!" for Fortran comments.  And probably delete the "..." line.

File: ompi/mpi/man/man3/MPI_Cart_sub.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+function `MPI_Cart_sub` can be used to partition the communicator group
+into subgroups that form lower-dimensional Cartesian subgrids, and to
+build for each subgroup a communicator with the associated subgrid
+Cartesian topology. (This function is closely related to
+`MPI_Comm_split`.)
+
+Example: Assume that `MPI_Cart_create( ..., comm)` has defined a (2 x
+3 x 4) grid. Let `remain_dims` = (true, false, true). Then a call to
+
+    MPI_Cart_sub(comm, remain_dims, comm_new)
```
Comment (jsquyres) [inline ompi/mpi/man/man3/MPI_Cart_sub.md:63]: Quote this as C code, and ditch the indenting.

**Comment by jsquyres** [comment]
@Colton-K Can you remove that merge commit?  Rebase on top of master, if necessary.

**Comment by ibm-ompi** [comment]
The IBM CI (GNU/Scale) build failed! Please review the log, linked below.

 Gist: https://gist.github.com/2fd26e047f2c8ce202dc20c7cc482fba

**Comment by ibm-ompi** [comment]
The IBM CI (XL) build failed! Please review the log, linked below.

 Gist: https://gist.github.com/a80a54cfdfe4521d629f03b62404e823

**Review by jsquyres — APPROVED** [review]

**Pull request merged**